frontend/src/hooks/useBooks.ts

```diff
@@ -26,9 +26,17 @@ export default function useBooks() {
             .catch(console.error)
     }
 
+    function deleteBook(id: string) {
+        return axios.delete("/api/books/" + id)
+            .then(() => {
+                loadAllBooks()
```

Comment:
This additional network request is not necessary if you remove the deleted book from the "books" array in the frontend.
For example like this` setBooks(books.filter(book=>book.id!==id)).`